crates/chia-bls/src/cached_bls.rs

```diff
@@ -107,12 +109,18 @@ impl BLSCache {
     #[pyo3(name = "aggregate_verify")]
     pub fn py_aggregate_verify(
         &mut self,
-        pks: &PyList,
-        msgs: &PyList,
+        pks: &pyo3::Bound<PyList>,
+        msgs: &pyo3::Bound<PyList>,
         sig: &Signature,
     ) -> PyResult<bool> {
-        let pks_r = pks.iter().map(|item| item.extract::<PublicKey>().unwrap());
-        let msgs_r = msgs.iter().map(|item| item.extract::<&[u8]>().unwrap());
+        let pks_r = pks
+            .iter()
+            .unwrap()
+            .map(|item| item.unwrap().extract::<PublicKey>().unwrap());
```

Comment:
I suppose there used to be an `unwrap()` here before your change, but now there are two. I would expect we prefer to raise a python exception instead of panic if something's wrong. Do you feel confident that `unwrap()` is the best choice?

Reply:
Didn't you say panics get converted to Python errors? I suppose it's just cleaner to return an actual error anyways though, I'll look into it

Reply:
yeah, they do. but panics don't propagate gracefully through rust code